### PR TITLE
DEC-626: Fix views

### PR DESCRIPTION
**Pull request by jgondron**
All of the data layer changes have been made. These changes are needed to fix the controllers and views to work with those data changes. I also made changes to the behavior of the site setup form. On save, it will now redirect back to the current form for editing. This was taking the user back to the top form, which meant the user would have to click on the form again to see the changes that they made.

### CODE REVIEW

**Comment by jonhartzler** [comment]
:+1:

File: spec/services/save_honeypot_image_spec.rb [hunk 1]
```diff
@@ -112,4 +117,16 @@
       expect(subject.send(:api_url)).to eq("http://localhost:3019")
     end
   end
+
+  context "group id" do
```
Comment (rfox2) [inline spec/services/save_honeypot_image_spec.rb]: What is the group id used for?

**Comment by rfox2** [comment]
:+1:

File: spec/services/save_honeypot_image_spec.rb (hunk 1 above)
Comment (jgondron) [inline spec/services/save_honeypot_image_spec.rb]: Its just a logical grouping of images on the honeypot side. We use collection id as that grouping id.

**Pull request merged**